tools/check_install.py

```diff
@@ -101,17 +99,6 @@ def main(args):
             logging.warning("--> %s is not installed.\n###### Raw Error ######\n%s#######################" % (name, traceback.format_exc()))
             is_correct_installed_list.append(False)
 
-    # warp-rnnt was only tested and successfull with CUDA_VERSION=10.0
-    # however the library installation is optional ("warp-transducer" is used by default)
```

Comment:
Why you removed

Reply:
Because you made `warp-transducer `and` warp-rnnt `optional and `warp-transducer` was already removed from checking. Should I keep it?

Reply:
Basically, I carefully wrote it. Please keep.

Reply:
Sorry I'm a bit confused with this one. I further modified to have the whole picture but I'll revert after I understood:

 1) `warp-transducer` (and also `warp-ctc`) checking was already removed before this PR  (lines ares commented)
 1) What I removed are only additions/modifications I introduced in #2444. It concers `warp-transducer` + `warp-rnnt` and some special cases for them: checking in case ESPnet was installed in CPU mode + manual appending to `library_list`(l.106 - l.113) and checking version through dpkg for both rnnt lib because they forgot adding `__version__` in some cases (l.134 - L.141).

Reply:
This is temporary state. Please keep.

Reply:
OK, file reverted.